website/src/components/Header.tsx

```diff
@@ -109,7 +109,7 @@ export default class Header extends React.Component<HeaderProps, any> {
                                         </li>
                                         <li className="header__nav-item nav-item" role="presentation">
                                             <a
-                                                className="header__nav-link nav-link"
+                                                className="header__nav-link nav-link btn btn-primary mb-2"
```

Comment:
Is the `mb-2`needed? It looks better aligned without it: 
With:
![image](https://user-images.githubusercontent.com/16265452/80562572-a62d7080-8a1a-11ea-9170-a32f16410353.png)


Without:
![image](https://user-images.githubusercontent.com/16265452/80562593-b6dde680-8a1a-11ea-8d7f-106eb61f9f6e.png)

Reply:
```suggestion
                                                className="header__nav-link nav-link btn btn-primary"
```

Reply:
agree - thanks!